Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -248,6 +264,20 @@ void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
   }
 }
 
+void MenuBar::UpdateToolsMenu(bool emulation_started)
```

Comment:
Yes. NANDContentManager will be removed in the future, so we should start using the proper replacement in new code.
It's equivalent, just more reliable because there is no caching involved (there shouldn't be). And it makes checking if the system menu is actually installed in the future easier.